diff --git a/scripts/capstone/_install.sh b/scripts/capstone/_install.sh
index ae8d435..2b3413c 100644
--- a/scripts/capstone/_install.sh
+++ b/scripts/capstone/_install.sh
@@ -5,7 +5,7 @@
 . $SCRIPTS_DIR/capstone/_envsetup.sh
 
 _download() {
-	REPO="git@github.com:capstone-engine/capstone.git"
+	REPO="https://github.com/capstone-engine/capstone.git"
 	__git_clone "$REPO" "$CAPSTONE_PATH" "$CAPSTONE_VERSION"
 }
 
diff --git a/scripts/cmake/_install.sh b/scripts/cmake/_install.sh
index 3dd06fd..fb32fdb 100644
--- a/scripts/cmake/_install.sh
+++ b/scripts/cmake/_install.sh
@@ -5,7 +5,7 @@
 . $SCRIPTS_DIR/cmake/_envsetup.sh
 
 _download() {
-	REPO="git@github.com:Kitware/CMake.git"
+	REPO="https://github.com/Kitware/CMake.git"
 	__git_clone $REPO $CMAKE_PATH $CMAKE_VERSION
 }
 
diff --git a/scripts/dwarves/_install.sh b/scripts/dwarves/_install.sh
index 1907b2b..062f529 100644
--- a/scripts/dwarves/_install.sh
+++ b/scripts/dwarves/_install.sh
@@ -5,7 +5,7 @@
 . $SCRIPTS_DIR/dwarves/_envsetup.sh
 
 _download() {
-	REPO="git@github.com:acmel/dwarves.git"
+	REPO="https://github.com/acmel/dwarves.git"
 	__git_clone "$REPO" "$DWARVES_PATH" "$DWARVES_VERSION"
 }
 
diff --git a/scripts/gcc/_install.sh b/scripts/gcc/_install.sh
index 59cf5c4..1368bac 100644
--- a/scripts/gcc/_install.sh
+++ b/scripts/gcc/_install.sh
@@ -5,7 +5,7 @@
 . "$SCRIPTS_DIR/gcc/_envsetup.sh"
 
 _download() {
-	REPO="git://gcc.gnu.org/git/gcc.git"
+	REPO="https://gcc.gnu.org/git/gcc.git"
 	_GCC_VERSION="$(echo $GCC_VERSION | sed "s/_/\//")"
 	__git_clone "$REPO" "$GCC_PATH" "$_GCC_VERSION"
 	(cd $GCC_PATH; ./contrib/download_prerequisites)
diff --git a/scripts/linux/_install.sh b/scripts/linux/_install.sh
deleted file mode 100644
index d10b19d..0000000
--- a/scripts/linux/_install.sh
+++ /dev/null
@@ -1,25 +0,0 @@
-#!/bin/sh -e
-
-[ -n "$__RAZZERV2_READY" ] || exit 1
-
-. $SCRIPTS_DIR/linux/_envsetup.sh
-
-_download() {
-	LINUX="$KERNELS_DIR/linux"
-	if [ -f "$LINUX/.git" ]; then
-		return 0
-	fi
-	git submodule update --init "$LINUX"
-}
-
-_build() {
-	# Don't want to build a kernel at this time
-	:
-}
-
-_install() {
-	# Don't want to install a kernel at this time
-	:
-}
-
-_target="linux-submodule"
diff --git a/scripts/meson/_install.sh b/scripts/meson/_install.sh
index 9df8c6d..93abe8a 100644
--- a/scripts/meson/_install.sh
+++ b/scripts/meson/_install.sh
@@ -5,7 +5,7 @@
 . "$SCRIPTS_DIR/meson/_envsetup.sh"
 
 _download() {
-	REPO="git@github.com:mesonbuild/meson.git"
+	REPO="https://github.com/mesonbuild/meson.git"
 	__git_clone "$REPO" "$MESON_PATH" "$MESON_VERSION"
 }
 
diff --git a/scripts/ninja/_install.sh b/scripts/ninja/_install.sh
index a74a4da..65b524c 100644
--- a/scripts/ninja/_install.sh
+++ b/scripts/ninja/_install.sh
@@ -5,7 +5,7 @@
 . "$SCRIPTS_DIR/ninja/_envsetup.sh"
 
 _download() {
-	REPO="git@github.com:ninja-build/ninja.git"
+	REPO="https://github.com/ninja-build/ninja.git"
 	__git_clone "$REPO" "$NINJA_PATH" "$NINJA_VERSION"
 }
 
diff --git a/tools/qemu/capstone b/tools/qemu/capstone
index f8b1b83..9e27c51 160000
--- a/tools/qemu/capstone
+++ b/tools/qemu/capstone
@@ -1 +1 @@
-Subproject commit f8b1b833015a4ae47110ed068e0deb7106ced66d
+Subproject commit 9e27c51c2485dd37dd3917919ae781c6153f3688
diff --git a/tools/qemu/dtc b/tools/qemu/dtc
index 85e5d83..7da5d10 160000
--- a/tools/qemu/dtc
+++ b/tools/qemu/dtc
@@ -1 +1 @@
-Subproject commit 85e5d839847af54efab170f2b1331b2a6421e647
+Subproject commit 7da5d106c7400fc02b0b5bee1b565b07895f5132
diff --git a/tools/qemu/meson b/tools/qemu/meson
index 776acd2..8e8abba 160000
--- a/tools/qemu/meson
+++ b/tools/qemu/meson
@@ -1 +1 @@
-Subproject commit 776acd2a805c9b42b4f0375150977df42130317f
+Subproject commit 8e8abba8f59a9bc4f0d4cb35ceddee5b4a8f04cb
diff --git a/tools/qemu/roms/SLOF b/tools/qemu/roms/SLOF
index 33a7322..9f1ed94 160000
--- a/tools/qemu/roms/SLOF
+++ b/tools/qemu/roms/SLOF
@@ -1 +1 @@
-Subproject commit 33a7322de13e9dca4b38851a345a58d37e7a441d
+Subproject commit 9f1ed94cce4ef8e1ab1cc36afd4230e26f385d56
diff --git a/tools/qemu/roms/edk2 b/tools/qemu/roms/edk2
index 06dc822..2d69507 160000
--- a/tools/qemu/roms/edk2
+++ b/tools/qemu/roms/edk2
@@ -1 +1 @@
-Subproject commit 06dc822d045c2bb42e497487935485302486e151
+Subproject commit 2d69507a4dde02f1abf20c7eb3a43d1d3ef6b98f
diff --git a/tools/qemu/roms/ipxe b/tools/qemu/roms/ipxe
index 4bd064d..5c49e0a 160000
--- a/tools/qemu/roms/ipxe
+++ b/tools/qemu/roms/ipxe
@@ -1 +1 @@
-Subproject commit 4bd064de239dab2426b31c9789a1f4d78087dc63
+Subproject commit 5c49e0a3b9c0f8003eea2a378a563c02593e835a
diff --git a/tools/qemu/roms/openbios b/tools/qemu/roms/openbios
index 4a00411..c3a19c1 160000
--- a/tools/qemu/roms/openbios
+++ b/tools/qemu/roms/openbios
@@ -1 +1 @@
-Subproject commit 4a0041107b8ef77e0e8337bfcb5f8078887261a7
+Subproject commit c3a19c1e54977a53027d6232050e1e3e39a98a1b
diff --git a/tools/qemu/roms/opensbi b/tools/qemu/roms/opensbi
index 234ed8e..a280764 160000
--- a/tools/qemu/roms/opensbi
+++ b/tools/qemu/roms/opensbi
@@ -1 +1 @@
-Subproject commit 234ed8e427f4d92903123199f6590d144e0d9351
+Subproject commit a2807646a85878cb631cbc2b012cf888fc4427d7
diff --git a/tools/qemu/roms/qboot b/tools/qemu/roms/qboot
index a5300c4..8ca302e 160000
--- a/tools/qemu/roms/qboot
+++ b/tools/qemu/roms/qboot
@@ -1 +1 @@
-Subproject commit a5300c4949b8d4de2d34bedfaed66793f48ec948
+Subproject commit 8ca302e86d685fa05b16e2b208888243da319941
diff --git a/tools/qemu/roms/qemu-palcode b/tools/qemu/roms/qemu-palcode
index bf0e136..99d9b4d 160000
--- a/tools/qemu/roms/qemu-palcode
+++ b/tools/qemu/roms/qemu-palcode
@@ -1 +1 @@
-Subproject commit bf0e13698872450164fa7040da36a95d2d4b326f
+Subproject commit 99d9b4dcf27d7fbcbadab71bdc88ef6531baf6bf
diff --git a/tools/qemu/roms/seabios b/tools/qemu/roms/seabios
index 155821a..b686f46 160000
--- a/tools/qemu/roms/seabios
+++ b/tools/qemu/roms/seabios
@@ -1 +1 @@
-Subproject commit 155821a1990b6de78dde5f98fa5ab90e802021e0
+Subproject commit b686f4600792c504f01929f761be473e298de33d
diff --git a/tools/qemu/roms/seabios-hppa b/tools/qemu/roms/seabios-hppa
index 73b740f..3391c58 160000
--- a/tools/qemu/roms/seabios-hppa
+++ b/tools/qemu/roms/seabios-hppa
@@ -1 +1 @@
-Subproject commit 73b740f77190643b2ada5ee97a9a108c6ef2a37b
+Subproject commit 3391c580960febcb9fa8f686f9666adaa462c349
diff --git a/tools/qemu/roms/sgabios b/tools/qemu/roms/sgabios
index cbaee52..72f39d4 160000
--- a/tools/qemu/roms/sgabios
+++ b/tools/qemu/roms/sgabios
@@ -1 +1 @@
-Subproject commit cbaee52287e5f32373181cff50a00b6c4ac9015a
+Subproject commit 72f39d48bedf044e202fd51fecf3e2218fc2ae66
diff --git a/tools/qemu/roms/skiboot b/tools/qemu/roms/skiboot
index 3a6fded..d365a01 160000
--- a/tools/qemu/roms/skiboot
+++ b/tools/qemu/roms/skiboot
@@ -1 +1 @@
-Subproject commit 3a6fdede6ce117facec0108afe716cf5d0472c3f
+Subproject commit d365a01a0996136fae75c3080800be14632c98e8
diff --git a/tools/qemu/roms/u-boot b/tools/qemu/roms/u-boot
index d368926..3833600 160000
--- a/tools/qemu/roms/u-boot
+++ b/tools/qemu/roms/u-boot
@@ -1 +1 @@
-Subproject commit d3689267f92c5956e09cc7d1baa4700141662bff
+Subproject commit 3833600dbaa2d7008063f3127abf21be8f0fe6cd
diff --git a/tools/qemu/roms/vbootrom b/tools/qemu/roms/vbootrom
index 0c37a43..c076c6b 160000
--- a/tools/qemu/roms/vbootrom
+++ b/tools/qemu/roms/vbootrom
@@ -1 +1 @@
-Subproject commit 0c37a43527f0ee2b9584e7fb2fdc805e902635ac
+Subproject commit c076c6bc0b74bff38f43bf28d9df2ff9bde55ca8
diff --git a/tools/qemu/slirp b/tools/qemu/slirp
index 8f43a99..19b8f26 160000
--- a/tools/qemu/slirp
+++ b/tools/qemu/slirp
@@ -1 +1 @@
-Subproject commit 8f43a99191afb47ca3f3c6972f6306209f367ece
+Subproject commit 19b8f2645cf71fc58781fcf4f04046a546138872
diff --git a/tools/qemu/tests/fp/berkeley-softfloat-3 b/tools/qemu/tests/fp/berkeley-softfloat-3
index b64af41..a0c6494 160000
--- a/tools/qemu/tests/fp/berkeley-softfloat-3
+++ b/tools/qemu/tests/fp/berkeley-softfloat-3
@@ -1 +1 @@
-Subproject commit b64af41c3276f97f0e181920400ee056b9c88037
+Subproject commit a0c6494cdc11865811dec815d5c0049fba9d82a8
diff --git a/tools/qemu/tests/fp/berkeley-testfloat-3 b/tools/qemu/tests/fp/berkeley-testfloat-3
index 5a59dce..e7af975 160000
--- a/tools/qemu/tests/fp/berkeley-testfloat-3
+++ b/tools/qemu/tests/fp/berkeley-testfloat-3
@@ -1 +1 @@
-Subproject commit 5a59dcec19327396a011a17fd924aed4fec416b3
+Subproject commit e7af9751d9f9fd3b47911f51a5cfd08af256a9ab
diff --git a/tools/qemu/ui/keycodemapdb b/tools/qemu/ui/keycodemapdb
index 6119e6e..ab223f5 160000
--- a/tools/qemu/ui/keycodemapdb
+++ b/tools/qemu/ui/keycodemapdb
@@ -1 +1 @@
-Subproject commit 6119e6e19a050df847418de7babe5166779955e4
+Subproject commit ab223f5d31134b6b518dc18b7bd62374d87748ce
